ci-operator/config/stolostron/acmqe-autotest/stolostron-acmqe-autotest-main__vboulos-acm-ocp4.12-lp-interop.yaml

```
@@ -0,0 +1,74 @@
+base_images:
+  acmqe-grc-test:
+    name: "2.7"
+    namespace: acm-qe
+    tag: acmqe-grc-test
+  application-ui-test:
+    name: "2.7"
+    namespace: acm-qe
+    tag: application-ui-test
+  clc-ui-e2e:
+    name: "2.7"
+    namespace: acm-qe
+    tag: clc-ui-e2e
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  fetch-managed-clusters:
+    name: autotest
+    namespace: acm-qe
+    tag: fetch-managed-clusters
+  observability-core-automation:
+    name: "2.7"
+    namespace: acm-qe
+    tag: observability-core-automation
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.12"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: acm-interop-aws
+  cron: 0 1 * * 1
```

```suggestion
  cron: 0 6 * * 1
```
Because this is UTC time and we want to run at 1 am we should make this change.